### ISSUE TITLE
Port examples to fs2-cats

### ISSUE THREAD

**Issue by rossabaker** [http4s/http4s#866]
A lot of it is commented.  Part of the work here is figuring out what we can uncomment as progress has been made in other areas.

The current compilation failure is:

```
[error] http4s/examples/src/main/scala/com/example/http4s/ExampleService.scala:65: value valueOr is not a member of org.http4s.ParseResult[org.http4s.Uri]
[error]       TemporaryRedirect(uri("/http4s/"))
```

This is probably as simple as changing the URI macro definition.

**Comment by rossabaker** [comment]
#879 fixes the compilation error.  There is still an opportunity for people to uncomment and test more of the examples.

**Comment by aeons** [comment]
Fixed by #1384.
